run-clang-format.py

```diff
@@ -143,7 +144,8 @@ def run_clang_format_diff_wrapper(args, file):
     except DiffError:
         raise
     except Exception as e:
-        raise UnexpectedError("{}: {}: {}".format(file, e.__class__.__name__, e), e)
+        raise UnexpectedError("{}: {}: {}".format(
```

Comment:
Can we use f-strings here?

Reply:
It's not what I intended, and I reset the commit